docs/development.md

```diff
@@ -0,0 +1,24 @@
+# Development
+
+For setting up the development environment, we rely on the
+[devcontainer feature](https://code.visualstudio.com/docs/remote/containers) of VS Code
+in combination with Docker Compose.
+
+To use it, you have to have Docker Compose as well as VS Code with its "Remote - Containers"
+extension (`ms-vscode-remote.remote-containers`) installed.
+Then open this repository in VS Code and run the command
+`Remote-Containers: Reopen in Container` from the VS Code "Command Palette".
+
+This will give you a full-fledged, pre-configured development environment including:
+- infrastructural dependencies of the service (databases, etc.)
+- all relevant VS Code extensions pre-installed
+- pre-configured linting and auto-formatting
+- a pre-configured debugger
+- automatic license-header insertion
+
+Moreover, inside the devcontainer, a command `dev_install` is available for convenience.
+It installs the service with all development dependencies, and it installs pre-commit.
+
+The installation is performed automatically when you build the devcontainer. However,
+if you update dependencies in the [`./pyproject.toml`](./pyproject.toml) or the
+[`./requirements-dev.txt`](./requirements-dev.txt), please run it again.
```

Comment:
Both links are broken

Reply:
Development section moved back to the readme template. It is not imported anymore.